src/sections/greenHeader.html

```diff
@@ -0,0 +1,32 @@
+<header class="fixed flex items-start justify-between w-full px-3 pt-4 lg:px-8 ">
```

Comment:
remove unnecessary spaces in your classes
```suggestion
<header class="fixed flex items-start justify-between w-full px-3 pt-4 lg:px-8">
```

Reply:
Done!